packages/ui-components/tailwind.config.ts

```diff
@@ -2,7 +2,7 @@ import type { Config } from 'tailwindcss';
 import plugin from 'tailwindcss/plugin';
 
 export default {
-  content: ['./**/*.tsx'],
+  content: ['./Common/**/*.tsx', './Containers/**/*.tsx', './Icons/**/*.tsx'],
```

Comment:
Interesting. But why?

Reply:
Tailwind wasn't happy that the glob included node_modules, so I figured I'd might as well fix it